test/process_group_rpc_agent_test_fixture.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import torch.distributed.rpc as rpc
+
+
+class ProcessGroupRpcAgentTestFixture(object):
+    @property
+    def rpc_backend_name(self):
+        return "PROCESS_GROUP"
+
+    @property
+    def rpc_agent_options(self):
+        return rpc.backend_registry.construct_rpc_agent_options(
+            self.rpc_backend,
```

Comment:
Where is `self.rpc_backend` initialized?

Reply:
In `RpcAgentTestFixture`.